scripts/gui.lua

```diff
@@ -1469,6 +1619,25 @@ function GUI.updateTrackerConfig(trackerInfo, tracker)
             blFlow["tlbe-tracker-left"].style = style
         elseif tracker.type == "player" then
             trackerInfo["tracker-untilbuild"].state = tracker.untilBuild
+        elseif tracker.type == "cityblock" then
+            local cityBlock = tracker.cityBlock
+            if cityBlock == nil then
+                return
+            end
+            
+            local sizeFlow = trackerInfo["cityblock-size"]
+            sizeFlow["tlbe-tracker-cityblock-size-x"].text = string.format("%d", cityBlock.blockSize.x)
+            sizeFlow["tlbe-tracker-cityblock-size-y"].text = string.format("%d", cityBlock.blockSize.y)
+
+            local offsetFlow = trackerInfo["cityblock-offset"]
+            offsetFlow["tlbe-tracker-cityblock-offset-x"].text = string.format("%d", cityBlock.blockOffset.x)
+            offsetFlow["tlbe-tracker-cityblock-offset-y"].text = string.format("%d", cityBlock.blockOffset.y)
+
+            local blockFlow = trackerInfo["cityblock-block"]
+            blockFlow["tlbe-tracker-cityblock-currentblock-x"].text = string.format("%d", cityBlock.currentBlock.x)
+            blockFlow["tlbe-tracker-cityblock-currentblock-y"].text = string.format("%d", cityBlock.currentBlock.y)
+
+            trackerInfo["tlbe-tracker-cityblock-blockScale"].text = string.format("%.1f", cityBlock.blockScale)
```

Comment:
We might need 2 decimals, to have a more precise options to set the scale (the bigger the block size is, the more precision is required to have a small padding)

Reply:
Done. I agree that it is a little odd to not be able to chose 1.25 scaling.